src/main.rs

```diff
@@ -94,6 +97,25 @@ impl App {
         }
     }
 
+    fn save_regions(&mut self, path: PathBuf) {
+        let mut f = File::create(path).expect("");
+        let regions_vm = &self.vm.slots[self.vm.index].regions_vm;
+        let mut text = String::from("");
+        for (region, (activated, _, _,_)) in regions_vm.regions.iter() {
+            if *activated {
+                // Matches the format provided by a CE table for Elden Ring
+                text.push_str(&REGIONS.lock().unwrap()[region].0.to_string());
+                text.push_str(&String::from("\n"));
+            }
+        }
+        let res = f.write_all(&text.as_bytes());
+
+        match res {
```

Comment:
This doesn't do anything? and the err will crash the tool because of the `todo!`